zenodo/config.py

```diff
@@ -1043,26 +1043,42 @@ def _(x):
 #: Defined facets for records REST API.
 RECORDS_REST_FACETS = dict(
     records=dict(
+            default_aggs=['access_right', 'type', 'file_type', 'keywords'],
         aggs=dict(
             type=dict(
-                terms=dict(field="resource_type.type"),
+                terms=dict(field='resource_type.type'),
                 aggs=dict(
                     subtype=dict(
-                        terms=dict(field="resource_type.subtype"),
+                        terms=dict(field='resource_type.subtype'),
                     )
                 )
             ),
             access_right=dict(
-                terms=dict(field="access_right"),
+                terms=dict(field='access_right'),
             ),
             file_type=dict(
-                terms=dict(field="filetype"),
+                terms=dict(field='filetype'),
             ),
             keywords=dict(
-                terms=dict(field="keywords"),
+                terms=dict(field='keywords'),
             ),
             communities=dict(
-                terms=dict(field="communities"),
+                terms=dict(field='communities'),
+            ),
+            related_identifiers=dict(
+                terms=dict(field='related.resource_type.type'),
+                aggs=dict(
+                    related_identifiers_subtype=dict(
+                        terms=dict(field='related.resource_type.subtype'),
+                    )
+                )
+        ),
+            publication_date=dict(
+                date_histogram=dict(
+                    field='publication_date',
+                    interval='1y',
+                    format='yyyy-MM-dd'
```

Comment:
```suggestion
                    format='yyyy'
```

Wouldn't that be easier for the display as well?